### PR TITLE
LOG-4757: update version to 5.9

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
https://issues.redhat.com/browse/LOG-4713

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,13 +43,13 @@ export CURRENT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD;)
 export IMAGE_TAG?=127.0.0.1:5000/openshift/origin-$(OPERATOR_NAME):$(CURRENT_BRANCH)
 BUNDLE_TAG=$(error set OVERLAY to deploy or run a bundle)
 
-export LOGGING_VERSION?=5.8
+export LOGGING_VERSION?=5.9
```
Comment (xperimental) [inline Makefile:46]: `OPENSHIFT_VERSIONS` in this file needs a bump to `v4.13-v4.17`. Looks fine otherwise :+1:

**Comment by xperimental** [comment]
Hm. I think the e2e test might be right for a change ... There's no 5.9 version of EO (and as far as we currently know: there will not be one). I think we might need a change to the script that installs EO for the e2e test and make it use 5.8 even when on the 5.9 `master` branch.

**Comment by jcantrill** [comment]
> Hm. I think the e2e test might be right for a change ... There's no 5.9 version of EO (and as far as we currently know: there will not be one). I think we might need a change to the script that installs EO for the e2e test and make it use 5.8 even when on the 5.9 `master` branch.

change made.  I came to the same conclusion

**Comment by jcantrill** [comment]
Likely blocked by https://github.com/openshift/cluster-logging-operator/pull/2225

**Pull request merged**